src/blocks/plugins/conditions/edit.js

```diff
@@ -401,6 +415,24 @@ const Edit = ({
 		setAttributes({ otterConditions });
 	};
 
+	/**
+	 * Toggle the value of the condition in the nested array.
+	 *
+	 * @param {any} value The value to set.
+	 * @param {number} index The index of the group.
+	 * @param {number} key The index of the condition.
+	 * @param {string} type The type of the condition.
+	 */
+	const toggleValueInArray = ( value, index, key, type ) => {
+		const otterConditions = [ ...attributes.otterConditions ];
+		if ( otterConditions[ index ][ key ][ type ].includes( value ) ) {
```

Comment:
This will fail given the type isn't set. We need to add it to the default or at least do a check before this.